libevm/VM.cpp

```diff
@@ -15,12 +15,78 @@
     along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+#include "interpreter.h"
 #include "VM.h"
 
 using namespace std;
 using namespace dev;
```

Comment:
These can be deleted since you explicitly define everything in the `namespase dev { namespace eth {` below?